src/lolite/lib/hooks/PythonScript.py

```diff
@@ -0,0 +1,13 @@
+from lolite.lib.hooks.hook_base import HookBase
+
+
+class Hook(HookBase):
+
+
+    def __init__(self, logger, arguments):
+        super().__init__(logger, arguments)
+
+    def execute_hook(self):
+        self._logger.debug(f"Running Python Hook: {self._arguments}")
+        if not self._subproc.run_command_exit_code(f"python3 {self._arguments}") == 0:

```

Comment:
Might be worth parameterizing or providing a way to define the version. Some python versions can be 3.x which may cause this to not work.